template.json

```diff
@@ -287,7 +287,7 @@
     },
     "gear": {
       "templates": ["gear"],
-      "bonus": 0
+      "bonus": ""
```

Comment:
yeah for calculation it would need an additional field and a new #each-function because one injury can have multiple effects.

But since it could be anything from nothing to 5 or so I think we have three major possibilities to do this.
- let it be a string like right now
would be the most flexible but no auto-calculations
- make for everything a field (health, mind, every attribute, skill etc)
would be fixed and like 22 fields but could be auto-calculated
- make something like the special feature from weapons/armors with to field (drop-down of health/mind/skills/attributes) and a modifier input. 
![image](https://user-images.githubusercontent.com/36819852/174680133-cb043f36-5dde-4b11-88d6-f92896181da3.png)

I think I like the 3. version the most. It would allow to handle that more flexible without hundreds of default-checkboxes.
But I have to think and try some things to get this work.^^